src/chrome/content/rules/BroadbandReports.xml

```diff
@@ -1,45 +0,0 @@
-<!--
-	Problematic domains:
-
-		- (www.)dslreports.com	(mismatched, CN: secure.dslreports.com)
-
-
-	Mixed content:
-
-		- Web bugs, on secure.dslreports.com from:
-
-			- www.reddit.com *
-			- badge.stumbleupon.com *
-
-	* Secured by us
-
--->
-<ruleset name="Broadband Reports">
-
-	<target host="broadbandreports.com" />
-	<target host="www.broadbandreports.com" />
-	<target host="*.dslr.net" />
-	<target host="dslreports.com" />
-	<target host="*.dslreports.com" />
-
-
-	<securecookie host="^\.dslr\.net$" name=".+" />
-
-
-	<!--	Redirect loop:
-
-		https://mail1.eff.org/pipermail/https-everywhere-rules/2012-October/001352.html
-												-->
-		<exclusion pattern="^http://(?:www\.)?(?:broadband|dsl)reports\.com/forum/[a-qs-z][_a-z]+(?:,[a-z]+|,?\d+)?(?:~[=a-z]+)?$" />
```

Comment:
Looks like you got rid of the exclusion patterns. Did you confirm that these are no longer necessary?